### DISCUSSION TITLE
Disabling display overdrive using G-Helper

### DISCUSSION THREAD

**Discussion by Spencer-1S** [seerge/g-helper#2329] [Q&A]
I have an ASUS TUF F15 (FX507ZV4).
core i7-12700H, RTX 4060, 144Hz (100%sRGB & g-sync capable) FHD inbuilt-display

Backstory (please read):
Whenever I play games, I can see a weird white trail along the edges of moving structures (it's called overshoot I think). Eventually I came to know that this is happening due to excess display overdrive on my laptop screen. In some games it's intensity is more, in some it is very less (but it is there). Note that this occurred only when I enabled G-sync but when I disabled G-sync, motion blur starts occurring (little). I contacted NVIDIA and after 1 month of chat and emails, they also said that this is happening due to excess overdrive on my laptop screen. They asked me to contact ASUS for providing a way to control overdrive. 

![Monitor-Response-Time-Explained-DisplayNinja-1024x254](https://github.com/seerge/g-helper/assets/107408307/bad1baee-b7e2-4a0d-bd62-2908e5207d35)

By default my ASUS laptop does not have any option to control overdrive, so having this question put up on reddit, I came to know about G-helper. With great hopes in my mind, I disabled the screen overdrive using G-helper but unfortunately nothing happened. There is no visual change and inverse ghosting (overshoot) still persists.

I began to think, **_if g-helper was actually able to disable overdrive or not? Does this function works for my laptop?_**

At this point I am feeling hopeless. Please help.

**Comment by seerge** [comment]
@Spencer-1S hello. Is that your post on reddit  https://www.reddit.com/r/Asustuf/comments/1bpi9og/how_to_reduce_display_overdrive_od_on_my_asus_tuf/ ? 

Cause if so - I can repeat same thing as I wrote to you there

It's some settings in a game or video driver :) **Overdrive is a physical property of your screen, it won't be visible on screen-record at all**
(same if you would crack your screen in a center, and then make a screenshot - crack won't be on a screenshot)

**Reply by Spencer-1S** [reply]
Thank you very much for the reply (both places).
Okay, I will proceed on to investigate further (game settings and drivers), considering that G-helper has disabled overdrive for me.

Again, thank you very much for your time :)

**Reply by Spencer-1S** [reply]
@seerge My device is not listed ( ASUS TUF F15 - FX507ZV4 ) on the supported devices list of G-Helper. Could that be the reason that disabling overdrive _might_ not be working? Can you please confirm?

Also, yes overdrive is a physical property of screen but excess of overdrive causes overshoot effect (also called inverse ghosting) which is visible on screen record (I think).

**Reply by seerge** [reply]
@Spencer-1S your device is listed, it's just TUF laptop.

Overdrive is NOT VISIBLE on any record obviously.  It is visible for your eyes, or if you film it with a (good quality) camera :)

**Reply by SpicySide** [reply]
I am also having the same issue with a 2024 Scar 4090 with miniled. Using Armoury Crate I have no issues, after removing it and installing ghelper it almost seems like extreme motion blur is turned on.  Super noticeable in The Last of Us, I had to reinstall Armoury Crate. I wonder why the issue appears while using ghelper. Turning off overdrive doesn't appear to change anything, I wonder if overdrive is actually working.

**Reply by RohanAjayy** [reply]
@SpicySide would you mind telling me what solved your problem ? Im having the same situation in my strix 3050
